Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -1518,11 +1516,10 @@ String CSSPrimitiveValue::customCSSText() const
 
     CSSTextCache& cssTextCache = WebCore::cssTextCache();
 
-    if (m_hasCachedCSSText && m_cachedCSSTextUsesLegacyPrecision == s_useLegacyPrecision) {
```

Comment:
Can we remove:
* https://searchfox.org/wubkat/rev/f78fd4847fdb47a9159a81e971c8b2af61d8d197/Source/WebCore/css/CSSPrimitiveValue.cpp#55
* https://searchfox.org/wubkat/rev/f78fd4847fdb47a9159a81e971c8b2af61d8d197/Source/WebCore/css/CSSPrimitiveValue.h#267 ?

Reply:
Yes seems like we should to me as well.

Reply:
@nt1m Thanks! Done.